nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/resources/docs/org.apache.nifi.reporting.SiteToSiteStatusReportingTask/additionalDetails.html

```diff
@@ -105,14 +113,16 @@ <h2>Record writer</h2>
 	{ "name" : "queuedBytes", "type" : ["long", "null"]},
 	{ "name" : "backPressureBytesThreshold", "type" : ["long", "null"]},
 	{ "name" : "backPressureObjectThreshold", "type" : ["long", "null"]},
+	{ "name" : "backPressureDataSizeThreshold", "type" : ["string", "null"]},
 	{ "name" : "isBackPressureEnabled", "type" : ["string", "null"]},
 	
     // fields for processors
 	{ "name" : "processorType", "type" : ["string", "null"]},
 	{ "name" : "averageLineageDurationMS", "type" : ["long", "null"]},
 	{ "name" : "flowFilesRemoved", "type" : ["long", "null"]},
 	{ "name" : "invocations", "type" : ["long", "null"]},
-	{ "name" : "processingNanos", "type" : ["long", "null"]}
+	{ "name" : "processingNanos", "type" : ["long", "null"]},
+	{ "name" : "counters", "type": { "type": "map", "values": "string" }}
```

Comment:
@mattyb149 should executionNode also be included?

Reply:
Yes I missed that one, thanks! Will update